cuda_bindings/cuda/bindings/_path_finder/load_nvidia_dynamic_library.py

```diff
@@ -21,6 +27,21 @@
     )
 
 
+def _load_other_in_subprocess(libname, error_messages):
+    code = f"""\
+from cuda.bindings._path_finder.load_nvidia_dynamic_library import load_nvidia_dynamic_library
+import json
+import sys
+loaded = load_nvidia_dynamic_library({libname!r})
+sys.stdout.write(json.dumps(loaded.abs_path, ensure_ascii=True))
```

Comment:
Q: Are there concerns dumping the path directly to stdout without json? I suppose we can handle unicode paths just fine?

Reply:
This is moot now that we no longer use `_load_anchor_in_subprocess`